spec/controllers/graders_controller_spec.rb

```diff
@@ -346,6 +346,14 @@
           @ta1 = create(:ta)
           @ta2 = create(:ta)
           @ta3 = create(:ta)
+
+          # Making two new TA's that are 'inactive', to test additional functionality
+          @ta4 = create(:ta)
```

Comment:
you can pass in `hidden: true` to avoid setting the attribute manually and then saving.

(and throughout the new tests you added)